src/dialogs/config-flow/step-flow-pick-handler.ts

```diff
@@ -69,6 +74,26 @@ class StepFlowPickHandler extends LitElement {
         slug: handler,
       }));
 
+      // Get Supported Brands in a digestable list of objects
+      const supportedBrands: SupportedBrandObj[] = [];
+      for (const [domain, domainBrands] of Object.entries(h.supportedBrands)) {
+        for (const [slug, name] of Object.entries(domainBrands)) {
+          const index = supportedBrands.findIndex((b) => b.slug === slug);
+
+          // Replace the array if exists, to be changed to append to array after supporting multiple supported brands
+          if (index !== -1) {
+            supportedBrands[index].supported_flows = [domain];
+            continue;
+          }
+
+          supportedBrands.push({
```

Comment:
Push it directly  to integrations
```suggestion
          integrations.push({
```